### PR TITLE
DCOS-37545 Correcting nav titles, formatting, removed TOC.

### PR DESCRIPTION
**Pull request by SarahStegall**
## Description
https://jira.mesosphere.com/browse/DCOS-37545

## Urgency
- [ ] Blocker 
- [ ] High
- [x] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

File: pages/services/nifi/0.1.0-1.5.0/index.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
 layout: layout.pug
-navigationTitle:  DSE 2.0.2-5.1.2
-title: DSE 2.0.2-5.1.2
+navigationTitle: NiFi 0.1.0 - 1.5.0
```
Comment (ryadav88) [inline pages/services/nifi/0.1.0-1.5.0/index.md]: remove the space in version. should be `NiFi 0.1.0-1.5.0`

File: pages/services/nifi/0.1.0-1.5.0/index.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
+title: NiFi 0.1.0 - 1.5.0
```
Comment (ryadav88) [inline pages/services/nifi/0.1.0-1.5.0/index.md]: `NiFi 0.1.0-1.5.0`

Comment (ryadav88) [reply pages/services/nifi/0.1.0-1.5.0/index.md]: would apply to all references of the version in the docs

File: pages/services/nifi/index.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
 ---
 layout: layout.pug
-navigationTitle:  Apache NiFi
-title: DC/OS Apache NiFi
+navigationTitle:
+title: NiFi
```
Comment (ryadav88) [inline pages/services/nifi/index.md]: would this fix the name here as well? https://docs.mesosphere.com/services/

Should just be `NiFi` on https://docs.mesosphere.com/services/

**Comment by SarahStegall** [comment]
Merging per Ravi Yadav.

**Pull request merged**